frontend/src/lib/lemon-ui/Tooltip/Tooltip.tsx

```diff
@@ -158,7 +159,14 @@ export function Tooltip({
                             {typeof title === 'function' ? title() : title}
                             {docLink && (
                                 <p className={`mb-0 ${title ? 'mt-1' : ''}`}>
-                                    <Link to={docLink} target="_blank" className="text-xs">
+                                    <Link
+                                        to={docLink}
+                                        target="_blank"
+                                        className="text-xs"
+                                        data-ph-capture-attribute-autocapture-event-name="clicked tooltip doc link"
+                                        data-ph-capture-attribute-doclink={docLink}
+                                        onClick={() => posthog.capture('clicked tooltip doc link', { docLink })}
```

Comment:
Dont really need this because of autocapture, right?

Reply:
Yeah I forgot I can just make an action out of the autocapture one to get a more discrete event, updated